test/checks/keyboard/landmark-is-top-level.js

```diff
@@ -23,6 +23,16 @@ describe('landmark-is-top-level', function() {
 		assert.deepEqual(checkContext._data, { role: 'main' });
 	});
 
+	it('should return false if the complementary landmark is in another landmark', function() {
```

Comment:
Also, just had a thought, what happens if the first `header` element after `body` element happens with in `shadowDOM`?

Question being, do we need to consider `shadowDOM` for this rule?

Reply:
The check already has Shadow DOM coverage.